index.html

```diff
@@ -767,24 +767,68 @@ <h1>Streaming GCN<br>Notices in Python</h1>
     <section>
       <div class="image-and-caption">
         <div>
-          <img src="img/screen-shot-gcn-markdown.png" width="700" alt="Screen shot of GCN Circular web form">
+          <img src="img/screen-shot-circulars.png" width="700" alt="Screen shot of GCN Circulars archive">
         </div>
         <div data-markdown>
           <textarea data-template>
-            ## Modernization of GCN Circulars (Coming Soon!)
-            * Manage your own subscriptions
-            * Self-register to submit using peer endorsement
-            * Submit circulars by web form, API, or email
-            * New searchable archive of Circulars
-            * Planned enhancements
-              * "Astro-Flavored Markdown"
-              * Automatically mint DOIs and BiBTeX entries
-              * Link your ORCID to your GCN Circulars
+            ## Modernization of GCN Circulars (Transition April 17, 2023)
+            * Browse and search Circulars in our all-new archive.
+            * Sign up for and manage your own email subscriptions.
+            * Enroll yourself and your colleagues to submit Circulars with arXiv-style
+              peer endorsements for new contributors.
+            * Submit Circulars with our new Web form, or continue to submit by email.
           </textarea>
         </div>
       </div>
     </section>
-    
+
+    <section data-markdown>
+      <textarea data-template>
+        ## Here is what you need to know about preparing for the transition on April 17.
```

Comment:
remove 'here is'

April 17 is redundant!

Reply:
For tomorrow, it's worth reminding people over and over that the transition is on April 17.  For the webinars, we'll definitely want to remove it, as you suggest.